### PR TITLE
[Dashboard] Remove chakra-ui Flex (2)

### PR DESCRIPTION
**Pull request by kien-ngo**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
This PR updates various components in the dashboard to replace `<Flex>` with `<div>` for better styling flexibility.

### Detailed summary
- Replaced `<Flex>` with `<div>` in multiple components for styling flexibility.

> The following files were skipped due to too many changes: `apps/dashboard/src/components/mission/MajorSection.tsx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by kien-ngo** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4699?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#4699** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4699?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#4697** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4697?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kien-ngo and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/dashboard/src/components/mission/DesireSection.tsx [hunk 1]
```diff
@@ -19,7 +13,7 @@ const DesireSection = () => {
       >
         thirdweb will...
       </Heading>
-      <Flex maxW="2xl">
```
Comment (MananTank) [inline apps/dashboard/src/components/mission/DesireSection.tsx:22]: Are we sure that chakra's 2xl is same as tailwind 2xl. 

It doesn't look like that's the case 
chakra - 96em  (1536px)
https://v2.chakra-ui.com/docs/styled-system/responsive-styles

tailwind - 42rem; (672px)
https://tailwindcss.com/docs/max-width

File: apps/dashboard/src/components/mission/MajorSection.tsx [hunk 2]
```diff
@@ -19,41 +13,41 @@ const MajorSection = () => {
       >
         However, there are two major obstacles to mass adoption.
       </Heading>
-      <Flex maxW="2xl">
```
Comment (MananTank) [inline apps/dashboard/src/components/mission/MajorSection.tsx:22]: same thing here

**Review by MananTank — CHANGES_REQUESTED** [review]
please address the max-w changes

File: apps/dashboard/src/components/mission/DesireSection.tsx (hunk 1 above)
Comment (kien-ngo) [reply apps/dashboard/src/components/mission/DesireSection.tsx:22]: oops, going over the other changes. ty

Comment (kien-ngo) [reply apps/dashboard/src/components/mission/DesireSection.tsx:22]: This belongs to a marketing page so probably just gonna hard-code it to 96rem.

**Review by MananTank — APPROVED** [review]

File: apps/dashboard/src/components/mission/DesireSection.tsx (hunk 1 above)
Comment (MananTank) [reply apps/dashboard/src/components/mission/DesireSection.tsx:22]: sounds good, I think we don't even need to make changes in marketing pages - we'll move them to framer

**Pull request merged**